src/server/pfs/fuse/server_test.go

```diff
@@ -1026,8 +1026,9 @@ func TestProjects(t *testing.T) {
 
 		require.NoError(t, json.NewDecoder(resp.Body).Decode(&projectData))
 		require.Equal(t, len(projectData), 3)
-		require.Equal(t, projectData[0].Project.Name, emptyProjectName)
+		// todo(fahad): I had to reverse the order of these, is that okay?
```

Comment:
I think this is expected, because our new projects table automatically sorts the projects in the order they are created.